docs/dev_guide/deploy-feathr-api-as-webapp.md

```diff
@@ -6,6 +6,8 @@ parent: Developer Guides
 
 # Feathr REST API
 
+> :warning: This document is out of date and will be updated in the future.
```

Comment:
This doc is already replaced by https://github.com/feathr-ai/feathr/blob/main/docs/dev_guide/build-and-push-feathr-registry-docker-image.md and can be removed